.github/workflows/contracts.yml

```diff
@@ -134,22 +144,20 @@ jobs:
 
       - uses: google-github-actions/setup-gcloud@v0.2.0
         with:
-          project_id: ${{ secrets.GOOGLE_PROJECT_ID }}
+          project_id: ${{ env.GOOGLE_PROJECT_ID }}
           service_account_key: ${{ secrets.KEEP_TEST_GCR_JSON_KEY }}
 
       - name: Fetch external contracts artifacts
         env:
-          CONTRACT_DATA_BUCKET: ${{ secrets.KEEP_TEST_CONTRACT_DATA_BUCKET }}
```

Comment:
Just to confirm, don't we need to set this variable?

Reply:
This variable will be loaded during `Load environment variables` step, from `ropsten.env` config file. I've tested that variables loaded during that step are available in later steps' shells as environment variables (this is true within a job, the variables are not shared between jobs).

Reply:
This variable will be loaded during `Load environment variables` step, from `ropsten.env` config file. I've tested that variables loaded during that step are available in later steps' shells as environment variables (this is true within a job, the variables are not shared between jobs).